app/helpers/view_helper.rb

```diff
@@ -146,6 +146,14 @@ def formatted_percentage(count, total)
     number_to_percentage(percentage, precision: precision)
   end
 
+  def protect_against_mistakes
+    if session[:confirmed_environment_at] && session[:confirmed_environment_at] > 10.seconds.ago #5.minutes.ago
+      yield
+    else
+      govuk_link_to '🦺 Confirm environment to make changes', support_interface_confirm_environment_path(from: request.fullpath)
```

Comment:
Not sure we need an emoji for this… and can cause screenreader issues too if we don’t describe it properly.